src/control/server/storage/bdev/backend_test.go

```diff
@@ -1332,3 +1334,75 @@ func TestBackend_Prepare(t *testing.T) {
 		})
 	}
 }
+
+func TestBdev_spdkBackend_ReadConfig(t *testing.T) {
+	for name, tc := range map[string]struct {
+		setup   func(t *testing.T, req *storage.BdevReadConfigRequest)
+		req     storage.BdevReadConfigRequest
+		expResp *storage.BdevReadConfigResponse
+		expErr  error
+	}{
+		"empty config path": {
+			req:    storage.BdevReadConfigRequest{},
+			expErr: errors.New("empty SPDK config path"),
+		},
+		"bad config path": {
+			req: storage.BdevReadConfigRequest{
+				ConfigPath: "/bad/path",
+			},
+			expErr: errors.New("open /bad/path: no such file or directory"),
+		},
+		"good config path; bad config": {
```

Comment:
how about a case with a well-formed config with bad fields?

Reply:
Sure, I can add one just for completeness, although again, this seems like we're testing the json package.